qa/os/src/test/java/org/elasticsearch/packaging/util/Distribution.java

```diff
@@ -84,6 +88,46 @@ public boolean isPackage() {
         }
     }
 
+    public static class PackagingConditional {
```

Comment:
I realize I suggested this class to you in a previous PR comment, but if the only place we need it is inside PackagingTest, I'm not sure it is worthwhile over using switch statements.